### ISSUE TITLE
[bug]: Sidebar issue in mobile view <SheetTitle> should be added in line 222.

### ISSUE THREAD

**Issue by Arunbalaji07** [shadcn-ui/ui#6284]
### Describe the bug

intercept-console-error.ts:40 DialogContent requires a DialogTitle for the component to be accessible for screen reader users.

If you want to hide the DialogTitle, you can wrap it with our VisuallyHidden component.}

### Affected component/components

Sidebar

### How to reproduce

1. In sidebar.tsx file add the below line

<SheetTitle className="sr-only">Mobile Sidebar</SheetTitle>

### Logs

```bash
hook.js:600 DialogContent requires a DialogTitle for the component to be accessible for screen reader users.

If you want to hide the DialogTitle, you can wrap it with our VisuallyHidden component.

For more information, see https://radix-ui.com/primitives/docs/components/dialog Error Component Stack
    at _c1 (sheet.tsx:57:6)
    at Sidebar (sidebar.tsx:178:13)
    at AppSidebar (AppSidebar.tsx:28:28)
    at div (<anonymous>)
    at div (<anonymous>)
    at SidebarProvider (sidebar.tsx:60:13)
    at div (<anonymous>)
    at DashboardLayout [Server] (<anonymous>)
    at ThemeProvider (theme-provider.tsx:7:5)
    at QueryClientContextProvider (query-client-provider.tsx:13:5)
    at ClerkProvider [Server] (<anonymous>)
    at body (<anonymous>)
    at html (<anonymous>)
    at RootLayout [Server] (<anonymous>)


hook.js:608 Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. Error Component Stack
    at DescriptionWarning (Dialog.tsx:531:66)
    at Dialog.tsx:383:13
    at Dialog.tsx:258:58
    at Presence (Presence.tsx:12:11)
    at DialogContent (Dialog.tsx:233:64)
    at SlotClone (Slot.tsx:61:11)
    at Slot (Slot.tsx:13:11)
    at Primitive.div (Primitive.tsx:38:13)
    at Portal (Portal.tsx:22:22)
    at Presence (Presence.tsx:12:11)
    at Provider (createContext.tsx:59:15)
    at DialogPortal (Dialog.tsx:145:11)
    at _c1 (sheet.tsx:57:6)
    at Provider (createContext.tsx:59:15)
    at Dialog (Dialog.tsx:52:5)
    at Sidebar (sidebar.tsx:178:13)
    at AppSidebar (AppSidebar.tsx:28:28)
    at div (<anonymous>)
    at div (<anonymous>)
    at Provider (createContext.tsx:59:15)
    at TooltipProvider (Tooltip.tsx:68:5)
    at SidebarProvider (sidebar.tsx:60:13)
    at div (<anonymous>)
    at Provider (createContext.tsx:59:15)
    at TooltipProvider (Tooltip.tsx:68:5)
    at DashboardLayout [Server] (<anonymous>)
    at InnerLayoutRouter (layout-router.tsx:319:3)
    at RedirectErrorBoundary (redirect-boundary.tsx:43:5)
    at RedirectBoundary (redirect-boundary.tsx:74:36)
    at HTTPAccessFallbackErrorBoundary (error-boundary.tsx:49:5)
    at HTTPAccessFallbackBoundary (error-boundary.tsx:154:3)
    at LoadingBoundary (layout-router.tsx:454:3)
    at ErrorBoundary (error-boundary.tsx:183:3)
    at InnerScrollAndFocusHandler (layout-router.tsx:177:1)
    at ScrollAndFocusHandler (layout-router.tsx:294:3)
    at RenderFromTemplateContext (render-from-template-context.tsx:7:30)
    at OuterLayoutRouter (layout-router.tsx:507:3)
    at _ (index.mjs:1:843)
    at J (index.mjs:1:724)
    at ThemeProvider (theme-provider.tsx:7:5)
    at QueryClientProvider (QueryClientProvider.tsx:30:3)
    at QueryClientContextProvider (query-client-provider.tsx:13:5)
    at SWRConfig (config-context-client-x_C9_NWC.mjs:533:13)
    at OrganizationProvider (contexts.tsx:42:3)
    at ClerkContextProvider (ClerkContextProvider.tsx:20:11)
    at ClerkProviderBase (ClerkProvider.tsx:11:11)
    at Hoc (useMaxAllowedInstancesGuard.tsx:28:5)
    at ClerkNextOptionsProvider (NextOptionsContext.tsx:18:11)
    at NextClientClerkProvider (ClerkProvider.tsx:50:11)
    at ClientClerkProvider (ClerkProvider.tsx:127:11)
    at ClerkProvider [Server] (<anonymous>)
    at body (<anonymous>)
    at html (<anonymous>)
    at RootLayout [Server] (<anonymous>)
```

### System Info

```bash
I didn't find any relevant information.
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by Arunbalaji07** [comment]
<SheetTitle className="sr-only">Mobile Sidebar</SheetTitle>  this is the fix in line 222 on sidebar.tsx

**Comment by realS3BI** [comment]
Also I can recommend:
```tsx
<VisuallyHidden>
  <SheetHeader>
    <SheetTitle>Sidebar</SheetTitle>
    <SheetDescription>Mobile version of the sidebar</SheetDescription>
  </SheetHeader>
</VisuallyHidden>
```
you might have to `npm install @radix-ui/react-visually-hidden`

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
